src/theme.js

```diff
@@ -118,17 +121,17 @@ function getTheme({ style, name }) {
       "tab.activeBorderTop": "#f9826c",
 
       "breadcrumb.foreground": primer.gray[5],
-      "breadcrumb.focusForeground": primer.gray[9],
+      "breadcrumb.focusForeground": foreground,
       "breadcrumb.activeSelectionForeground": primer.gray[6],
       "breadcrumbPicker.background": pick({ light: primer.gray[0], dark: "#2b3036" }),
 
-      "editor.foreground": primer.gray[9],
-      "editor.background": pick({ light: primer.white, dark: primer.gray[0] }),
+      "editor.foreground": foreground,
+      "editor.background": primer.gray[0],
```

Comment:
I think this should stay `primer.white`.

```suggestion
      "editor.background": pick({ light: primer.white, dark: primer.gray[0] }),
```